specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-06-02-preview/fleets.json

```diff
@@ -0,0 +1,741 @@
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "x-ms-enum": {
+            "name": "FleetProvisioningState",
+            "modelAsString": true
+          },
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "Creating",
+            "Deleting",
+            "Updating"
+          ],
+          "description": "The provisioning state of the last accepted operation."
+        }
+      }
+    },
+    "FleetHubProfile": {
+      "type": "object",
+      "description": "The FleetHubProfile configures the fleet hub. it is enabled by default and can be disabled by setting the hubMode property.",
+      "properties": {
+        "dnsPrefix": {
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "description": "DNS prefix used to create the FQDN for the fleet hub."
+        },
+        "fqdn": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The FQDN of the fleet hub."
+        },
+        "kubernetesVersion": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The Kubernetes version of the fleet hub."
+        }
+      }
+    },
+    "FleetListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Fleet"
+          },
+          "description": "The list of fleets."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next page of fleets.",
+          "readOnly": true
+        }
+      },
+      "description": "The response from the List Fleets operation."
+    },
+    "FleetMembersListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FleetMember"
+          },
+          "description": "The list of members in a given fleet."
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URL to get the next page of fleet members."
+        }
+      },
+      "description": "The response from the List FleetMembers operation."
+    },
+    "FleetPatch": {
+      "type": "object",
+      "description": "Properties of a fleet that can be patched.",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "description": "Resource tags."
+        }
+      }
+    },
+    "FleetMember": {
+      "type": "object",
+      "description": "A member of the fleet. It contains a reference to an existing Kubernetes cluster on Azure, which is a member of the fleet.",
```

Comment:
This is phrased sorta weird. Maybe just drop the "which is a member of the fleet" at the end? Feels like that's just repeating "A member of the fleet" at the beginning.